node/plugin/cmd/main.go

```diff
@@ -114,8 +116,33 @@ func pluginOps(ctx *cli.Context) {
 		return
 	}
 
+	s := strings.Split(config.Socket, ":")
```

Comment:
Can we make this a method (like ParseOperatorSocket) next to https://github.com/Layr-Labs/eigenda/blob/66c1e06c51c7aedb951ede024627e42af21a7a3b/core/state.go#L17?